static/js/components/EducationTab.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Button from 'react-mdl/lib/Button';
+import ProfileTab from "../util/ProfileTab";
+import { generateNewEducation } from "../util/util";
+import _ from 'lodash';
+import {
+  boundMonthField,
+  boundYearField,
+  saveAndContinue
+} from '../util/profile_edit';
+import {
+  toggleEducationLevel,
+  openEducationForm,
+  closeEducationForm
+} from '../actions';
+import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
+import { Grid, Cell, Switch, Dialog, DialogTitle, DialogContent, DialogActions, FABButton, Icon } from 'react-mdl';
```

Comment:
it seems like we usually put dependency imports first, followed by a line break and then imports from our own stuff, eg:

```js
import React from 'react';
import _ from 'lodash';

import ProfileTab from '../components/ProfileTab';
```

might make sense to stick with that here, especially when there's so much stuff being imported